### PR TITLE
Cleanup website.

### PR DESCRIPTION
**Pull request by varunpatro**

### CODE REVIEW

File: _data/coreteam_members.yml [hunk 1]
```diff
@@ -1,10 +1,8 @@
-- name: Vishnu
+- name: Varun
```
Comment (ngzhian) [inline _data/coreteam_members.yml]: the takeover as began...

File: about.md [hunk 2]
```diff
@@ -13,17 +13,17 @@ notes: |
...
 
-Once a year, we run a series of technical workshops called [hackerschool](http://school.nushackers.org/). We also run the Hack&amp;Roll hackathon in the second semester of the academic year. Our coreteam members contribute to events like LadyPy and Software Freedom Day.
+Every semester, we run a series of technical workshops called [hackerschool](http://school.nushackers.org/). We also run the Hack&amp;Roll hackathon in the second semester of the academic year. Our coreteam members contribute to events like LadyPy and Software Freedom Day.
 
 We maintain and release open source code for the NUS community (see: our [code page](/code/)). Students and staff who have built NUS-specific projects and can no longer maintain them may come to us to host and maintain their code.
 
-We currently maintain [Download@NUS](http://download.nus.edu.sg/). We have a small team of people working with the NUS Computer Centre to host scientific data. We also have a small team of hardware hackers who meet at Friday Hacks.
+We currently maintain [Download@NUS](http://download.nus.edu.sg/) We have a team working with the NUS Computer Centre to host linux mirrors.
```
Comment (ngzhian) [inline about.md]: hm are we still doing it?

File: fridayhacks.md [hunk 3]
```diff
@@ -17,15 +17,15 @@ notes: |
...
+<big>Our weekly Friday meetups are called Friday Hacks. They include one or two technical talks, followed by a hacking session. The idea is simple: get people to come share their hacks, research, or technical knowledge with the technical community in NUS.</big>
 
 We have up to 2 talks at each session, talks are usually 30-45 minutes long. Dinner is always served (usually pizza). Food is served at 6:30pm, talks start at 7pm.
 
 <img src="http://i.imgur.com/sEBVHCx.jpg" alt="Friday Hacks #71" width="525" height="350" />
 
 <h2>Speaker FAQ</h2>
 <strong>When and where do Friday Hacks happen?</strong><br>
-Friday Hacks happen every Friday at 6:30pm in Seminar Room 3, level 2 of Town Plaza, University Town, National University of Singapore.
+Friday Hacks happen every Friday at 6:30pm. Venue is at Seminar Room 3, level 2 of Town Plaza, University Town, National University of Singapore unless otherwise stated.
```
Comment (ngzhian) [inline fridayhacks.md]: don't need "Venue", can say "6:30pm, at Seminar Room 3"

File: _data/coreteam_members.yml [hunk 4]
```diff
@@ -1,10 +1,8 @@
...
   description: |
-    a CS student who's interested in too many things at once. You can learn
-    absolutely nothing more about him at his website <a
-    href="http://vishnuprem.com">vishnuprem.com</a>, but you can follow him
-    on <a href="http://twitter.com/burnflare">twitter</a>. He's contactable at
-    vishnu@nushackers.org
+    a Year 2 CS undergraduate interested in Programming Languages and Algorithms.
+    He frequents hackathons and keeps busy with his startup.
+    He's a varsity Ultimate Frisbee player and cycles around Singapore on weekends.
```
Comment (burnflare) [inline _data/coreteam_members.yml]: Append your email address to the end of the description here

File: about.md [hunk 5]
```diff
@@ -32,7 +32,7 @@ We believe that hacking is necessary for good innovation. (In fact, the best com
 (Though, honestly, most of the time we hack because we think it's fun).
 
 ###<a name="goal"></a>Goal
-Our long term goal is to build a healthy community of passionate hackers in NUS. We think that this benefits everyone: professors benefit because they are able to source for good programmers; startups and tech companies benefit because they are able to recruit from a central pool; students benefit because they get to meet and learn from like-minded peers (and get opportunities, i.e.: from professors and tech companies). We think we are 2, 3 years away from this goal.
```
Comment (jin) [inline about.md]: We met this goal? :)

**Comment by jin** [comment]
LGTM

File: about.md (hunk 2 above)
Comment (varunpatro) [inline about.md]: Need to update this in the future

**Pull request merged**